### ISSUE TITLE
lunarvim launch error

### ISSUE THREAD

**Issue by tnxz** [pkgxdev/pantry#2311]
`lvim` with tea script / `tea lvim`

```
E282: Cannot read from "/opt/lunarvim.org/v1.3.0/share/lunarvim/lvim/init.lua
```

**Comment by jhheider** [comment]
Looks like one of the scripts needs relocatability fixes.

Yup, `bin/lvim`:
```sh
#!/usr/bin/env bash

export NVIM_APPNAME="${NVIM_APPNAME:-"lvim"}"

export LUNARVIM_RUNTIME_DIR="${LUNARVIM_RUNTIME_DIR:-"/opt/lunarvim.org/v1.3.0/share/lunarvim"}"
export LUNARVIM_CONFIG_DIR="${LUNARVIM_CONFIG_DIR:-"/opt/lunarvim.org/v1.3.0/.config/lvim"}"
export LUNARVIM_CACHE_DIR="${LUNARVIM_CACHE_DIR:-"/opt/lunarvim.org/v1.3.0/.cache/lvim"}"

export LUNARVIM_BASE_DIR="${LUNARVIM_BASE_DIR:-"/opt/lunarvim.org/v1.3.0/share/lunarvim/lvim"}"

exec -a "$NVIM_APPNAME" nvim -u "$LUNARVIM_BASE_DIR/init.lua" "$@"
```

Should be easy enough.

**Comment by tnxz** [comment]
```
E282: Cannot read from "/Users/run/.tea/lunarvim.org/v1.3.0/share/lunarvim/lvim/init.lua"
```
```
$ ls -al ~/.tea/lunarvim.org/v1.3.0/share/lunarvim/lvim/init.lua
ls: ~/.tea/lunarvim.org/v1.3.0/share/lunarvim/lvim/init.lua: No such file or directory
```
```
$ ls -la ~/.tea/lunarvim.org/v1.3.0/share/lunarvim/lvim
~/.tea/lunarvim.org/v1.3.0/share/lunarvim/lvim -> /var/folders/y3/yr3bxsy95c3gfwwj61ytbp000000gq/T/8c9be7bf
```

**Comment by jhheider** [comment]
Weird use of a symlink. This should fix it: 8752b200feb95563230831f672623a853c5b5e01.
